profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_poetry/tmgmt_poetry_mock/tmgmt_poetry_mock.behat.inc

```diff
@@ -228,25 +228,19 @@ class TMGMTPoetryMockSubContext extends RawDrupalContext implements DrupalSubCon
    */
   public function assertTranslationRequestDocumentIsValidXhtml() {
 
-    if (!class_exists('tidy')) {
-      throw new Exception('You need Tidy extension to test if XHTML is valid.');
-    }
+    $msg = 'You need Tidy extension to test if XHTML is valid.';
+    assert(class_exists('tidy'), equals(TRUE), $msg);
```

Comment:
For me `equals(TRUE)` is really easy to understand.
But I will change that and let's move forward.